PurchasesTests/Support/ManageSubscriptionsHelperTests.swift

```diff
@@ -73,7 +73,10 @@ class ManageSubscriptionsHelperTests: XCTestCase {
     }
 
     func testShowManageSubscriptionsInIOS() throws {
-#if os(iOS)
+        guard #available(iOS 10.0, *) else {
+            throw XCTSkip("Not supported")
+        }
```

Comment:
doesn't this test still run on other platforms?

Reply:
ohh, I see, this test is a part of `StoreKitUnitTests`, which is iOS-only

Reply:
Could you move it into the `StoreKitUnitTests` folder to avoid confusion? I realize that this file was already there before this PR

Reply:
Oh yeah, I didn't realize it's in that target! I'll do that after you merge #1065.

Reply:
@aboedo done